lib/spack/spack/cmd/style.py

```diff
@@ -352,6 +349,22 @@ def run_black(black_cmd, file_list, args):
     return returncode
 
 
+@tool("fish_indent", suffix=".fish")
+def run_fish_indent(fish_indent_cmd, file_list, args):
+    if args.fix:
+        fish_indent_args = ["--write"]
+    else:
+        fish_indent_args = ["--check"]
+
+    output = fish_indent_cmd(*fish_indent_args, *file_list, fail_on_error=False, output=str)
+    returncode = fish_indent_cmd.returncode
+
+    rewrite_and_print_output(output, args)
```

Comment:
The output of `fish_indent --check` is a list of files that don't pass. Not sure if we want to do any fancier formatting.